File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # nums.sort()
        # for i in range(1 , len(nums)):
        #     if nums[i - 1] == nums[i]:
        #         return True
        # return False

         return len(nums) != len(set(nums))

        # seen = set()
        # for num in nums:
        #     if num in seen:
        #         return True
        #     seen.add(num)
        # return False

File: 0217-contains-duplicate/0217-contains-duplicate.c
int sorting(const void* a , const void* b){
    return (*(int*)a - *(int*)b);
}

bool containsDuplicate(int* nums, int numsSize) {
    qsort(nums , numsSize, sizeof(int), sorting);
    for(int i = 1; i < numsSize; i++){
        if(nums[i] == nums[i - 1]){
            return true;
        }
    }
    return false;
}